lib/ngx/ocsp.lua

```diff
@@ -109,16 +113,22 @@ function _M.validate_ocsp_response(resp, chain, max_errmsg_len)
     local sizep = get_size_ptr()
     sizep[0] = errbuf_size
 
-    local rc = C.ngx_http_lua_ffi_ssl_validate_ocsp_response(
-                        resp, #resp, chain, #chain, errbuf, sizep)
+    local rc = C.ngx_http_lua_ffi_ssl_validate_ocsp_response(resp, #resp,
+                                                             chain, #chain,
+                                                             errbuf, sizep,
+                                                             next_update_p)
 
     if rc == FFI_OK then
-        return true
+        local next_update = tonumber(next_update_p[0])
+        if next_update == 0 then
+          next_update = nil
+        end
+        return true, next_update
     end
 
     -- rc == FFI_ERROR
 
-    return nil, ffi_str(errbuf, sizep[0])
+    return nil, nil, ffi_str(errbuf, sizep[0])
```

Comment:
Please don't change the order of exsting return value because this will break backward-compatibility and 
break existing user code. Let use use the 3rd return value for this new data.

Also, will you please add some docs for this new feature?

Reply:
I tried to stick to the common convention of returning the error message as the last value. But if we want to avoid the backward incompatibility at the cost of breaking that convention, I can do it.

> Also, will you please add some docs for this new feature?

Definitely, I'll add some docs once we settle on the implementation.